src/pe_reports/stakeholder/views.py

```diff
@@ -535,7 +531,7 @@ def stakeholder():
     formExternal = InfoFormExternal()
 
     if formExternal.validate_on_submit():
-        logging.info("Got to the submit validate")
+        LOGGER.info("Got to the submit validate")
```

Comment:
Same as above regarding `info` vs. `debug`.

Reply:
@dav3r 
The logger category has been changed as you have mentioned. See commit [d11f89c](https://github.com/cisagov/pe-reports/pull/169/commits/d11f89c78939b26ed8344d0ad355b62febfc21d8)